{
    "Serilog": {
        "Using": ["Serilog.Sinks.Console"],
        "MinimumLevel": "Warning",
        "WriteTo": [{
                "Name": "Console",
                "Args": {
                    "formatter": "Serilog.Formatting.Compact.CompactJsonFormatter, Serilog.Formatting.Compact"
                }
            },
            {
                "Name": "File",
                "Args": {
                    "path": "logs/Health.txt",
                    "rollOnFileSizeLimit": true,
                    "rollingInterval": "Day",
                    "fileSizeLimitBytes": 1048576,
                    "retainedFileCountLimit": 7
                }
            }
        ],
        "Enrich": ["FromLogContext", "WithMachineName", "WithThreadId"],
        "Properties": {
            "Application": "Health"
        }
    },
    "HealthChecksUI": {
        "HealthChecks": [{
                "Name": "ApiGateway",
                "Uri": "http://api.nmro.local/hc"
            },
            {
                "Name": "Oidc",
                "Uri": "http://oidc.nmro.local/hc"
            }
        ],
        "Webhooks": [{
            "Name": "Slack",
            "Uri": "https://hooks.slack.com/services/T0GHRA4CR/BB551L6SW/LWYRVRA2NEs4pQG7pbPym",
            "Payload": "{\"text\":\"The HealthCheck [[LIVENESS]] is failing with the error message: [[FAILURE]]. [[DESCRIPTIONS]]. <http://yourappstatus|Click here> to get more details.\",\"channel\":\"#general\",\"link_names\": 1,\"username\":\"monkey-bot\",\"icon_emoji\":\":monkey_face:\"}",
            "RestoredPayload": "{\"text\":\"The HealthCheck [[LIVENESS]] is recovered. All is up and running\",\"channel\":\"#general\",\"link_names\": 1,\"username\":\"monkey-bot\",\"icon_emoji\":\":monkey_face:\"}"
        }],
        "EvaluationTimeinSeconds": 60,
        "MinimumSecondsBetweenFailureNotifications": 60
    },
    "ServiceDiscovery": {
        "DiscoveryAddress": "http://consul:8500",
        "ServicePort": "80"
    }
}
